symphony-bdk-core/src/test/java/com/symphony/bdk/core/activity/command/SlashCommandTest.java

```diff
@@ -44,6 +47,55 @@ void testSlashCommandWithBotMentionSuccess() {
     assertTrue(handlerCalled.get());
   }
 
+  @Test
+  void testSlashCommandsWithArgs() {
```

Comment:
Could we consider using [JUnit 5 Parametrized tests](https://www.baeldung.com/parameterized-tests-junit-5) here to test more use cases? (just a proposal)